core/src/database/franchise/player/player.model.ts

```diff
@@ -26,7 +26,13 @@ export class Player extends BaseModel {
     @Field(() => Int)
     skillGroupId: number;
 
-    @Column({type: "numeric"})
+    @Column({
+        type: "numeric",
+        transformer: {
+            from: (value: string) => parseFloat(value),
+            to: (value: number) => value.toString(),
+        },
+    })
```

Comment:
Why is this necessary again? Why don't we need this on other number fields?

Reply:
`numeric` is returned from typeorm as a string.

Reply:
Why don't we change the column type in the DB?

Reply:
Not sure. That's just what was there. I was going to change it to `double` or `float` but wasn't sure if there was a reason.

Reply:
Lets change the column type